packages/palette/src/elements/Input/Input.tsx

```diff
@@ -44,30 +47,24 @@ export const Input: React.ForwardRefExoticComponent<
   ) => {
     const [boxProps, inputProps] = splitBoxProps(rest)
 
+    const inputName = inputProps.name || "palette-input"
```

Comment:
`suggestion` It's possible we want to make this a required prop rather than a fallback. I don't really think there should be a fallback value though, either way.

Reply:
Actually, t's totally valid, of course, to have inputs without names, for instance when they are part of a non-form component — like an autocomplete UI or something.

Reply:
I'm concerned if we have multiple inputs on a page without names, would the labels be associated with the same input?

example:

```html
<input name="palette-input" ... />
<label for="palette-input" ...>

...

<input name="palette-input" ... />
<label for="palette-input" ...>

```

Is that even possible to have two inputs with the same name? and if so, should we add a random digit at the end of the fallback input name?

Reply:
Again: we really cannot have a default name here.

Reply:
Just removed the default name :)